syft/pkg/java_metadata.go

```diff
@@ -1,6 +1,19 @@
 package pkg
 
-import "github.com/package-url/packageurl-go"
+import (
+	"github.com/anchore/syft/internal"
+	"github.com/package-url/packageurl-go"
+)
+
+const PomPropertiesGroupIDJiraPlugins = "com.atlassian.jira.plugins"
+
+var JenkinsPluginPomPropertiesGroupIDs = []string{
```

Comment:
nit: Would be nice to name these two related things with a consistent phrase pattern (I'm good with either of the existing patterns winning): `PomPropertiesGroupIDJiraPlugins` vs. `JenkinsPluginPomPropertiesGroupIDs`

Reply:
I tried on `PomPropertiesGroupIDsJenkinsPlugins`  and `PomPropertiesGroupIDsJenkinsPlugin` for size, it seemed odd with the order and plurality. I can swap out `PomPropertiesGroupIDJiraPlugins` for `JiraPluginPomPropertiesGroupID`.